hq/app/logging/LogConfig.scala

```diff
@@ -98,6 +99,7 @@ object LogConfig {
             appender.setCredentialsProvider(buildCredentialsProvider(stsRole, config))
             appender.start()
             rootLogger.addAppender(appender)
+            rootLogger.setLevel(Level.INFO)
```

Comment:
I just had a look at another project and I'm pretty sure this is unnecessary - the change to logback.xml will modify the rootLogger

Reply:
Ok, thanks for looking further into this, I'll remove this bit